src/Command/Wishlist/AddProductToSelectedWishlist.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusWishlistPlugin\Command\Wishlist;
+
+use BitBag\SyliusWishlistPlugin\Entity\WishlistInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+
+final class AddProductToSelectedWishlist
```

Comment:
Select is in the name because already exist AddToWishlistAction which creates the default wishlist by clik on btn 'add to wishlist'. This action adding product to selecting wishlist in drop-down list when user create new wishlist (has more than 1)